src/ui/duke_script_runner.cpp

```diff
@@ -56,6 +57,8 @@ const auto GAME_SPEED_SLOT = 8;
 const auto INITIAL_SKILL_SELECTION = 1;
 const auto INITIAL_GAME_SPEED = 3;
 
+constexpr auto ANALOG_STICK_DEADZONE = 20'000;
```

Comment:
I am seeing this number format for the first time, what does having a ‘ represent here ?

Reply:
It's a handy little feature from C++ 14. The `'` has no meaning for the compiler, it just separates the digits for better readability :slightly_smiling_face: